catalogue/webapp/services/catalogue/fixtures/images-aggregations.ts

```diff
@@ -71,6 +74,7 @@ const aggregations: CatalogueResultsList<Image> = {
     },
     type: 'Aggregations',
   },
+  _requestUrl: 'https://api.wellcomecollection.org/catalogue/v2/images',
```

Comment:
This is to make the type checker happy; I'm not sure why it wasn't picked up before. 🤷‍♀️